pkg/networkmanager/dialogs-common.jsx

```diff
@@ -198,10 +201,27 @@ export const NetworkAction = ({ buttonText, iface, connectionSettings, type }) =
         if (type == 'vlan') settings = getVlanGhostSettings();
         if (type == 'team') settings = getTeamGhostSettings({ newIfaceName });
         if (type == 'bridge') settings = getBridgeGhostSettings({ newIfaceName });
+        if (type == 'wireguard') settings = getWireGuardGhostSettings({ newIfaceName });
     }
 
     const properties = { connection: con, dev, settings };
 
+    async function resolveDepsAndShowDialog(dlg) {
+        try {
+            await cockpit.script("command -v wg");
+            Dialogs.show(dlg);
+        } catch {
+            try {
+                const config = cockpit.manifests.network.config;
+                const os_release = await read_os_release();
+                await install_dialog(config.wireguard_package[os_release.ID]);
+                Dialogs.show(dlg);
+            } catch (e) {
+                console.error(e);
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230704-110812-1ea465d9-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)